### ISSUE TITLE
Class variables are not reset when reinitializing

### ISSUE THREAD

**Issue by pierbo** [cormas/cormas#54]
Hola,
Once a value has been assigned to a class var (for ex, Restrained harvestRate => 0.5), this value is not reset when reinitializing a new simulation (because of the lazzy initialization process).
in `initSimulation` of CormasModel, the instruction
	`self defaultAttributesSetter reApplyNewValues`
does nothing.
The `resetParameters` should be called to reset the class variables to there default values.

**Comment by SergeStinckwich** [comment]
I guess you talk about instance class variables ? They are only initialize ounce the class is loaded in the image or when you call ```initialize```.

You want to have instance class variables different for every simulation ?

**Comment by pierbo** [comment]
Yes, I was spoken about class variables (so I understand that they are called "instance class variables" in Pharo and defined as instanceVariableNames at the Class level...!).
Yes, as a class variable may be changed during a simulation (for ex., "catabolicRate" can increase because of temperature), it is necessary to automatically reset this variable to its default value at the beginning of a new simulation. As in Cormas we decided to use the lazzy initialization, so it should be reset to nil.
